eas_build_and_submit.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+sed -i 's/$IOS_USER_ID/${{ secrets.IOS_USER_ID }}/g' eas.json
```

Comment:
```suggestion
sed -i 's/$IOS_USER_ID/${IOS_USER_ID}/g' eas.json
```

shell, not GitHub action ;)

Reply:
haha good call!